be/src/exec/pipeline/exchange/local_exchange_sink_operator.cpp

```diff
@@ -24,6 +24,8 @@ Status LocalExchangeSinkOperator::prepare(RuntimeState* state) {
     RETURN_IF_ERROR(Operator::prepare(state));
     _exchanger->incr_sinker();
     _unique_metrics->add_info_string("ShuffleNum", std::to_string(_exchanger->source_dop()));
+    _peak_memory_usage_counter = _common_metrics->AddHighWaterMarkCounter(
```

Comment:
add to `_unique_metrics` not `_common_metrics`

Reply:
ok